instrumentation/grape/lib/opentelemetry/instrumentation/grape/instrumentation.rb

```diff
@@ -41,11 +34,12 @@ def gem_version
         end
 
         def require_dependencies
-          require_relative 'handler'
+          require_relative 'subscriber'
+          require_relative 'event_handler'
```

Comment:
You could also move this into subscriber

Reply:
done 😄